sw/device/sca/aes_serial.c

```diff
@@ -102,7 +109,7 @@ dif_aes_transaction_t transaction = {
     .key_provider = kDifAesKeySoftwareProvided,
     .mask_reseeding = kDifAesReseedPer8kBlock,
     .reseed_on_key_change = false,
-    .force_masks = false,
+    .force_masks = true,
```

Comment:
Nit: I wouldn't make this change because it even if one doesn't call the `aes_serial_seed_lfsr()` function now, e.g. because the capture code doesn't send that command, the masking is not turned off completely (the seed is non-zero) but the PRNG output is constant. This means the masking is not off but weaker. This might not be obvious from the traces. People might spend a lot of time analyzing not meaningful traces.

Reply:
Agree and reverted. So the default is now false and if the capture command seeds with 0 it is turned off as per the other addition.